meson/ldap/meson.build

```diff
@@ -21,7 +21,10 @@ if get_option('module-ldap') != 'disabled'
       endif
     endif
   endif
- 
+
+  if not dep_ldap_internal.found()
```

Comment:
nice. I wonder whether required: true on line 9 (and maybe 17?) would be simpler, or perhaps I am missing something.

Reply:
I think the execution than bails out early and the alternative method does not get run